docs/getting-started/tutorial/publish-the-changes-to-github/index.md

```diff
@@ -46,18 +46,18 @@ When you've finished staging all your changes, you are now ready to
 make the new commit:
 
 1. Type "Generate age histograms" where it says "Message" above the Commit button.
-   This messages summarises what your staged changes do.
+   This message summarises what your staged changes do.
 1. Click the Commit button or press ++ctrl+enter++
    to *commit* the staged changes to
    add them to the repository as stored in the codespace.
 
 ![Committing changes in GitHub.](../../../images/getting-started-codespaces-commit-message.png)
 
-## Push the changes to GitHub
+## Push the changes to the remote GitHub repository
```

Comment:
Although "remote" is the `git` terminology, I'm not repeating it here and on ln72 helps with clarity.